requirements/requirements.txt

```diff
@@ -17,9 +17,9 @@ pg8000
 ipykernel  # dev
 jupyter  # dev 
 transformers
-black
-flake8
-mypy
+black==22.3.0  # dev
```

Comment:
Modified version to match the GitHub Action's ones

Reply:
@alex-golts , we should move those to dev requirements (especially if we are forcing specific version)

Reply:
Cool, I agree